Makefile

```diff
@@ -121,15 +119,40 @@ save: $(DIST)
 	@echo IP=$(IP) >> $(CONFIG)
 	@echo UUID=$(UUID) >> $(CONFIG)
 	@echo ADAM_PORT=$(ADAM_PORT) >> $(CONFIG)
+	@echo EVE_SERIAL=$(EVE_SERIAL) >> $(CONFIG)
 	@echo EVE_BASE_REF=$(EVE_BASE_REF) >> $(CONFIG)
 	@echo EVE_BASE_VERSION=$(EVE_BASE_VERSION) >> $(CONFIG)
 	@echo ADAM_CA=$(ADAM_CA) >> $(CONFIG)
 	@echo EVE_CERT=$(EVE_CERT) >> $(CONFIG)
 	@echo LOGS=$(LOGS) >> $(CONFIG)
 
-eve_run: save eve_stop eve_live
+DEVICETREE_DTB_amd64=
+DEVICETREE_DTB_arm64=$(EVE_DIST)/dtb/eve.dtb
+DEVICETREE_DTB=$(DEVICETREE_DTB_$(ZARCH))
+
+# qemu settings
+QEMU_SYSTEM_arm64=qemu-system-aarch64
+QEMU_SYSTEM_amd64=qemu-system-x86_64
+QEMU_SYSTEM=$(QEMU_SYSTEM_$(ZARCH))
+
+QEMU_ACCEL_Y_Darwin=-M accel=hvf -cpu host
+QEMU_ACCEL_Y_Linux=-enable-kvm -cpu host
+QEMU_ACCEL_N_Darwin=-cpu SandyBridge
+QEMU_ACCEL_N_Linux=-cpu SandyBridge,vmx=on
+QEMU_ACCEL:=$(QEMU_ACCEL_$(ACCEL:%=Y)_$(shell uname -s))
+QEMU_OPTS_amd64= $(QEMU_ACCEL)
+QEMU_OPTS_arm64= -machine virt,gic_version=3 -machine virtualization=true -cpu cortex-a57 -machine type=virt
+QEMU_OPTS_COMMON= -display none -serial mon:stdio -readconfig $(DIST)/qemu.conf -nodefaults -no-user-config
+QEMU_OPTS=-smbios type=1,serial=$(EVE_SERIAL) $(QEMU_OPTS_COMMON) $(QEMU_OPTS_$(ZARCH))
+
+QEMU_CONF_OPTS_amd64=
+QEMU_CONF_OPTS_arm64=--dtb-part=$(dir $(DEVICETREE_DTB))
+QEMU_CONF_OPTS=$(QEMU_CONF_OPTS_$(ZARCH))
```

Comment:
I actually don't like this showing up directly in the Makefile. As I said -- the idea of hijacking EVE's makefile logic to run qemu is not the best one -- we should just own that part in Eden directly and we should do it through themplating.